utils.go

```diff
@@ -0,0 +1,58 @@
+package kinesis
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"text/template"
+
+	"github.com/gliderlabs/logspout/router"
+)
+
+type missingEnvVarError struct {
+	envVar   string
+	tmplName string
+}
+
+func (e *missingEnvVarError) Error() string {
+	return fmt.Sprintf("The %s template is missing. Please set the %s env variable\n", e.tmplName, e.envVar)
```

Comment:
This might be a little confusing - maybe just say:

`Missing required %s environment variable.`  Not sure what we get from naming the template, and it would almost make me think it was looking for a file or something and couldn't find it the way it's currently named.